### DISCUSSION TITLE
Welcome to built_with_flutter Discussions!

### DISCUSSION THREAD

**Discussion by aeiou-st** [aeiou-st/built_with_flutter#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mujammil-ios** [comment]
Ok got it it seems like you new to GitHub discussion so what will be next featire  in project

**Reply by aeiou-st** [reply]
Thanks it helps

**Comment by aeiou-st** [comment]
@Mujammil-ios thanks i'll note it down.
